<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VistaModificaProdottoAmministratore</class>
 <widget class="QWidget" name="VistaModificaProdottoAmministratore">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>499</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modifica Prodotto - CineMax</string>
  </property>
  <widget class="QLabel" name="Sfondo">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>791</width>
     <height>501</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    background: qlineargradient(
        x1: 0, y1: 1,
        x2: 0, y2: 0,
        stop: 0 #210A00,
        stop: 1 #7D2100
    );
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelIconaFotoAmministratore">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>30</y>
     <width>131</width>
     <height>131</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Immagini/IconaFotoAmministratore.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="labelNome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Nome:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelTitoloPrincipale">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>30</y>
     <width>351</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI Variable</family>
     <pointsize>20</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #D7320C;
}</string>
   </property>
   <property name="text">
    <string>Modifica Prodotto</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelIngredienti">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Ingredienti:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelDisponibile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>400</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Disponibilit√†:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelAllergeni">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>260</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Allergeni:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>330</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Prezzo:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditNome">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>130</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrezzo">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>360</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBoxDisponibile">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>430</y>
     <width>76</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QComboBox {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QComboBox QAbstractItemView {
    background-color: #320F00;
        color: #962D00;
    border: 1px solid #190700;
        border-radius: 4px;
    selection-background-color: #C83200;
}

QComboBox QAbstractItemView::item:hover {
    background-color: #C83200;
}</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllergene1">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>290</y>
     <width>91</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
        color: #962D00;
}

QCheckBox::indicator {
    background-color: #320F00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QCheckBox::indicator:hover {
            background-color: #C83200;
}

QCheckBox::indicator:checked {
            background-color: #C83200;
}


</string>
   </property>
   <property name="text">
    <string>Latticini</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllergene2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>290</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
        color: #962D00;
}

QCheckBox::indicator {
    background-color: #320F00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QCheckBox::indicator:hover {
            background-color: #C83200;
}

QCheckBox::indicator:checked {
            background-color: #C83200;
}


</string>
   </property>
   <property name="text">
    <string>Glutine</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllergene3">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>290</y>
     <width>121</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
        color: #962D00;
}

QCheckBox::indicator {
    background-color: #320F00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QCheckBox::indicator:hover {
            background-color: #C83200;
}

QCheckBox::indicator:checked {
            background-color: #C83200;
}


</string>
   </property>
   <property name="text">
    <string>Frutta a Guscio</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllergene4">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>290</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
        color: #962D00;
}

QCheckBox::indicator {
    background-color: #320F00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QCheckBox::indicator:hover {
            background-color: #C83200;
}

QCheckBox::indicator:checked {
            background-color: #C83200;
}


</string>
   </property>
   <property name="text">
    <string>Pesce</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllergene6">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>290</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
        color: #962D00;
}

QCheckBox::indicator {
    background-color: #320F00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QCheckBox::indicator:hover {
            background-color: #C83200;
}

QCheckBox::indicator:checked {
            background-color: #C83200;
}


</string>
   </property>
   <property name="text">
    <string>Uova</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditIngredienti">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>220</y>
     <width>511</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonConferma">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>440</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #961E00;
    color: #FF3C00;
    border: 2px solid #501400;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C83200;
}</string>
   </property>
   <property name="text">
    <string>Conferma</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButtonAnnulla">
   <property name="geometry">
    <rect>
     <x>560</x>
     <y>440</y>
     <width>91</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
    background-color: #961E00;
    color: #FF3C00;
    border: 2px solid #501400;
    border-radius: 6px;
}

QPushButton:focus {
    outline: none;
}

QPushButton:hover {
    background-color: #C83200;
}</string>
   </property>
   <property name="text">
    <string>Annulla</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxAllergene5">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>290</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QCheckBox {
        color: #962D00;
}

QCheckBox::indicator {
    background-color: #320F00;
    border: 1px solid #190700;
    border-radius: 4px;
}

QCheckBox::indicator:hover {
            background-color: #C83200;
}

QCheckBox::indicator:checked {
            background-color: #C83200;
}


</string>
   </property>
   <property name="text">
    <string>Soia</string>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreNome">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>130</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreIngredienti">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorePrezzo">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>360</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelErroreDisponibile">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>430</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="labelPrezzoPunti">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>330</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #C83C00;
}</string>
   </property>
   <property name="text">
    <string>Prezzo in Punti:</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEditPrezzoPunti">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>81</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLineEdit {
    background-color: #320F00;
    color: #962D00;
    border: 1px solid #190700;
    border-radius: 4px;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="labelErrorePrezzoPunti">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>360</y>
     <width>161</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QLabel {
    color: #FF0000
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
